index.html

```diff
@@ -966,7 +966,8 @@ <h2><dfn>Coalesced events</dfn></h2>
             <p>A <a>PointerEvent</a> has an associated <dfn>coalesced events list</dfn> (a list of
             zero or more <code>PointerEvent</code>s). For trusted <code>pointermove</code> and
             <code>pointerrawupdate</code> events, the list is a sequence of all <code>PointerEvent</code>s
-            that were coalesced into this event. For all other trusted event types, it is an empty list.
+            that were coalesced into this event, or if no events were coalesced, it contains a clone of the "parent" event itself.
```

Comment:
this is explicitly not talking about movement, since that seem to be separate issue altogether. Currently per specifications all the pointerevents should have movementX/Y 0 (whether or not the are in the coalesced events list). But the idea here is that this could work once movement is clarified elsewhere.